Ocelot.Downstream.Service.Instance2/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "Consul": {
+    "ConsulAddress": "http://localhost:8500",
+    "ServiceHost": "localhost",
```

Comment:
you can't use localhost, use hostname: instance1, instance2 instead to reflect your docker configuration,
as for Consul, use ConsulAddress: http://consul:8500